compiler/crates/relay-bin/src/main.rs

```diff
@@ -49,8 +58,8 @@ struct Opt {
     repersist: bool,
 
     /// Verbosity level
-    #[clap(long, arg_enum, default_value = "verbose")]
```

Comment:
A bit of a bummer here, I think the generated `--help` output will no-longer tell you what the default value is. Maybe there's some additional way to manually specify that with structopt?